doc/build.md

```diff
@@ -2,121 +2,193 @@
----                         | ---                                               | ---
-BUILD_MAYAUSD_LIBRARY       | Build Core USD libraries.                         | ON
-BUILD_ADSK_PLUGIN           | Build Autodesk USD plugin.                        | ON
-BUILD_PXR_PLUGIN            | Builds the Pixar USD plugin and libraries.        | ON
-BUILD_AL_PLUGIN             | Builds the Animal Logic USD plugin and libraries. | ON
-BUILD_TESTS                 | Build tests.                                      | ON
+##### Build Log
 
-# Building USD
+By default the build log is written into ```build_log.txt``` inside the build directory. If you want to redirect the output stream to the console instead,
+you can pass ```--redirect-outstream-file``` and set it to false.
 
-##### Flags, Version
+##### Additional flags and options
+
+Run the script with the ```--help``` parameter to see all the possible flags and short descriptions.
+
+#### 6. How To Run Unit Tests
+
+Unit tests can be run by setting ```--stages=test``` or by simply calling `ctest` directly from the build directory.
+
+For example, to run all Animal Logic's tests from the command-line go into ```build/<variant>/plugin/al``` and call `ctest`.
+
+```
+➜  ctest -j 8
+Test project /Users/sabrih/Desktop/workspace/build/Debug/plugin/al
+    Start 4: AL_USDMayaTestPlugin
+    Start 5: TestUSDMayaPython
+    Start 8: TestPxrUsdTranslators
+    Start 7: TestAdditionalTranslators
+    Start 1: AL_MayaUtilsTests
+    Start 3: Python:AL_USDTransactionTests
+    Start 2: GTest:AL_USDTransactionTests
+    Start 6: testMayaSchemas
+1/8 Test #2: GTest:AL_USDTransactionTests .....   Passed    0.06 sec
+2/8 Test #6: testMayaSchemas ..................   Passed    0.10 sec
+3/8 Test #3: Python:AL_USDTransactionTests ....   Passed    0.73 sec
+4/8 Test #1: AL_MayaUtilsTests ................   Passed    6.01 sec
+5/8 Test #8: TestPxrUsdTranslators ............   Passed    9.96 sec
+6/8 Test #5: TestUSDMayaPython ................   Passed   10.28 sec
+7/8 Test #7: TestAdditionalTranslators ........   Passed   12.06 sec
+8/8 Test #4: AL_USDMayaTestPlugin .............   Passed   27.43 sec
+100% tests passed, 0 tests failed out of 8
+```
+
+# Building Pixar USD
 
-When building Pixar USD, you need to pass ```--no-maya``` flag to ensure that ```third_party/maya``` plugin doesn't get built since this plugin is now part of maya-usd.
+##### Flags, Version
 
-It is important that the version between ```Pixar USD``` and ```Maya USD plugin```match. We are currently using ```v19.05``` for Maya plugin in ```master``` branch and therefore ```Pixar USD``` build should match this version.
+Pixar has recently removed support for building Maya USD libraries and plug-ins in their ```build_usd.py```. When building USD, it is important that the ```Pixar USD``` and ```Maya USD plug-in``` versions match. Consult the source versions table in the [Developer](DEVELOPER.md) document. 
```

Comment:
Is there a reason the version compatibility is still in a separate file? Wouldn't it make sense to have that in the compatibility list at the top of this file too?